less/_components/cards.less

```diff
@@ -0,0 +1,148 @@
+@card-panel-box-shadow: 4px 6px 9px 0px rgba(0,0,0,0.31);
+@card-panel-footer-font-size: 1.5em;
+@card-panel-footer-font-weight: 700;
+@card-panel-body-padding-top-none: 0px;
+@brand-primary-visited: darken(@brand-primary, 10); 
+
+@glyph-highlight-background-color: @brand-secondary;
+@glyph-highlight-container-padding: 10px 10px 5px 10px;
+@glyph-highlight-width: 75px;
+@glyph-highlight-height: 75px;
+@glyph-highlight-left-pos-mod: 90%;
+@glyph-highlight-font-color: @white;
+@glyph-highlight-font-size: 3em;
+
+.card-container {
+    & {
+      padding: @card-container-padding;
+    }
+    // handle linked cards
+    a .card-panel {
+      text-decoration: none;
+    }
+    a .card-panel:hover {
+      text-decoration: underline; 
+    }
+
+    .glyph-highlight {
+      & {
+        position: absolute;
+        left: calc(@glyph-highlight-left-pos-mod - @glyph-highlight-width);
+        top: 10px;
+        text-align: center;
+      }
+      .glyph-container {
+        width: @glyph-highlight-width;
+        height: @glyph-highlight-height;
+        font-size: @glyph-highlight-font-size;
+        color: @glyph-highlight-font-color;
+        background-color: @glyph-highlight-background-color;
+        padding: @glyph-highlight-container-padding;
+      }
+      .glyph-container img {
+        vertical-align: text-bottom;
+        max-width: 100%;
+        max-height: 100%;
+      }
+      .glyph-bottom {
+        height: 0;
+        width: 0;
+        border-left: calc(@glyph-highlight-width / 2) solid transparent;
+        border-right: calc(@glyph-highlight-width / 2) solid transparent;
+        border-bottom: calc(@glyph-highlight-width / 2) solid transparent;
+        border-top: calc(@glyph-highlight-width / 4) solid @glyph-highlight-background-color;
+      }
+    }
+    
+    .card-panel {
+      & {
+        border-radius: @card-panel-border-radius;
+        border: none;
+        overflow:hidden;
+        padding: 0;
+        height:100%;
+        box-shadow: @card-panel-box-shadow;
+      }
+  
+      &.horizontal-card {
+        @media (min-width: @screen-sm-min) {
+          & > * {
+            float: left;
+            min-height: 100%;
+          }
+          .panel-body, .panel-footer {
+            width: 100%;
+            padding-left: 25px;
+            padding-right: 25px;
+          }
+          .panel-heading ~ .panel-body, .panel-heading ~ .panel-footer {
+            width: 67%;
+            float: right;
+          }
+          .panel-heading {
+            width: 33%;
+            overflow-x: hidden;
+          }
+          .panel-heading .cover-image {
+            height: @card-image-max-height;
+            width: auto;
+            max-height: 100%;
+            max-width: 100%;
+          }
+          .panel-heading .cover-image.unbound {
+            height: 100%;
+          }
+        }
+      }
+      img.cover-image {
+        object-fit: cover;
+        width: 100%;
+        min-width: 100%;
```

Comment:
If the `min-width` equals to `width`, will it be any different if we only use `width: 100%` or `min-width:100%`?